vmware_fusion_mcp/vmware_client.py

```diff
@@ -0,0 +1,126 @@
+"""VMware Fusion REST API client."""
+
+import httpx
+from typing import Dict, List, Any
+from dataclasses import dataclass
+
+
+@dataclass
+class VMInfo:
+    """VM information from Fusion API."""
+
+    id: str
+    path: str
+    cpu: Dict[str, Any]
+    memory: Dict[str, Any]
+
+
+class VMwareClient:
+    """Client for VMware Fusion REST API."""
+
+    def __init__(
+        self,
+        base_url: str = "http://localhost:8697",
+        username: str = "",
+        password: str = "",
```

Comment:
I didn't see where they had been used; use `username` and `password` should be basic auth, continue implementing this part.